src/components/ebay-date-textbox/date-textbox.stories.ts

```diff
@@ -46,6 +46,13 @@ export default {
                 },
             },
         },
+        floatingLabel: {
+            type: "text|array",
+            control: { type: "object" },
+            description:
+                'If set then shows this text as the floating label. If separate floating labels are required for a range display, use an array of two strings (i. e. `["Start", "End"]`)..',
```

Comment:
Regarding the docs option, something like this would be much clearer:
```suggestion
                'If set then shows this text as the floating label (i. e. `"Single floating label"`). If separate floating labels are required for a range display, use an array of two strings (i. e. `["Range start floating label", "Range end floating label"]`)..',
```

Reply:
This helps avoid confusion. BTW, if we have something similar in other places, we may want to change those too. It's not clear that the string needs to be in quotes.